src/examples/banner/banner.tsx

```diff
@@ -0,0 +1,63 @@
+import { Component, Element, h, State } from '@stencil/core';
+
+@Component({
+    tag: 'limel-example-banner',
+    shadow: true,
+    styleUrl: 'banner.scss',
+})
+export class BannerExample {
+    @Element()
+    private host: HTMLElement;
+
+    @State()
+    private disabled = false;
+
+    private message = 'This is a non-blocking but also non-transient message';
+
+    private banner: HTMLLimelBannerElement;
+
+    constructor() {
+        this.toggleBanner = this.toggleBanner.bind(this);
+        this.someCustomAction = this.someCustomAction.bind(this);
+    }
+
+    public componentDidLoad() {
+        this.banner = this.host.shadowRoot.querySelector('limel-banner');
+    }
+
+    public render() {
+        return [
+            <limel-button
+                primary={true}
+                disabled={this.disabled}
+                label="Show banner"
+                onClick={this.toggleBanner}
+            />,
+            <limel-banner message={this.message} icon="exclamation_mark">
```

Comment:
I assume if we put this in some globally available element on the page, it'd survive navigation inside the app?

Reply:
In the webclient that is… 🙂

Reply:
yes, that's already tested here: https://github.com/Lundalogik/lime-webclient/pull/1075

Reply:
👌